imports/plugins/included/sms/server/methods/sms.js

```diff
@@ -49,7 +49,7 @@ Meteor.methods({
     check(shopId, String);
 
     const user = Accounts.findOne({ _id: userId });
-    const addressBook = user.profile.addressBook;
+    const addressBook = user ? user.profile.addressBook : false;
```

Comment:
Shouldn't we check for both `user` and `user.profile` before returning `addressBook`?

Reply:
I didn't really see the `profile.addressBook` nesting. I was in a rush to fix the profile of undefined error not knowing there will be another undefined of addressBook if profile is not available. I will do this check ASAP @zenweasel